generators/server/templates/src/test/resources/config/application-testcontainers.yml.ejs

```diff
@@ -0,0 +1,87 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+# ===================================================================
+# Spring Boot configuration.
+#
+# This configuration is used for unit/integration tests with testcontainers database containers.
+#
+# To activate this configuration launch integration tests with the 'testcontainers' profile
+#
+# More information on database containers: https://www.testcontainers.org/modules/databases/
+<%_ if (prodDatabaseType === 'mssql') { _%>
+#
+# You are required to accept EULA license for SQL server containers.
+# Follow the instructions at https://www.testcontainers.org/modules/databases/mssqlserver/
+<%_ } _%>
+<%_ if (prodDatabaseType === 'oracle') { _%>
+#
+# You have to specify an Oracle image name in a classpath file named testcontainers.properties.
+# Follow the instructions at https://www.testcontainers.org/modules/databases/oraclexe/
+<%_ } _%>
+# ===================================================================
+
+spring:
+    datasource:
```

Comment:
Could you add this block conditionally on `!reactive`, and add the `reactive` part you've done, event if it's not working yet?
It would help to understand the error and to fix it in a later PR.

Reply:
Also, it seems that `org.testcontainers:r2dbc` maven dependency need to be added in addition to the database module. Have you tried that?

Reply:
> Could you add this block conditionally on !reactive

I disabled the file generation when the application is reactive. 

> Also, it seems that org.testcontainers:r2dbc maven dependency need to be added in addition to the database module. Have you tried that?

Yes I did. I lost the changes I stashed when I tried to add reactive support, but I put the generated code here: https://github.com/ecostanzi/jhipster-r2dbc-testcontainers. It's a reactive+mysql example.